.circleci/config.yml

```diff
@@ -1,21 +1,12 @@
-# Ruby CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-ruby/ for more details
-#
-version: 2
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2
+
 jobs:
   build:
     docker:
-      # specify the version you desire here
-      - image: circleci/ruby:2.5
-      
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
-    working_directory: ~/repo
-
+      - image: circleci/ruby:2.6.3-stretch-node
```

Comment:
https://github.com/xflagstudio/zenform-rb/blob/7234113f2d3c7ed85f83514cce4998df9b9426e5/.ruby-version#L1

Please use [`circleci/ruby:2.5-xx` image](https://hub.docker.com/r/circleci/ruby/tags?page=1&name=2.5-) because zendesk-rb uses ruby 2.5.0.

Reply:
Fixed in 5fca28a